frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -27,64 +38,101 @@ export const StyledSidebar = styled.section(({ theme }) => ({
     left: 0,
     right: 0,
     bottom: 0,
-  },
-
-  // Scrollbars can look ugly, so we try to style them to look better
-  [`::-webkit-scrollbar-thumb:vertical,
-  ::-webkit-scrollbar-thumb:horizontal,
-  ::-webkit-scrollbar-thumb:vertical:active,
-  ::-webkit-scrollbar-thumb:horizontal:active`]: {
-    background: theme.colors.transparent,
-  },
-
-  "&:hover": {
-    [`::-webkit-scrollbar-thumb:vertical,
-    ::-webkit-scrollbar-thumb:horizontal`]: {
-      background: "rgba(0, 0, 0, 0.5)",
-    },
 
-    [`::-webkit-scrollbar-thumb:vertical:active,
-    ::-webkit-scrollbar-thumb:horizontal:active`]: {
-      background: "rgba(0, 0, 0, 0.61)",
-      borderRadius: "100px",
-    },
+    // // Hide scrollbar from nav area by putting this in front of that.
+    // "& ::-webkit-scrollbar": {
+    //   background: theme.colors.bgColor,
+    // },
   },
 }))
 
 export const StyledSidebarNavContainer = styled.div(({ theme }) => ({
   // TODO(vdonato): styling
   // * width of this component is 100% of the sidebar (probably needs
   //   adjustments to StyledSidebarContent)
+  position: "relative",
 }))
 
 export interface StyledSidebarNavItemsProps {
   expanded: boolean
+  hasSidebarElements: boolean
 }
 
 export const StyledSidebarNavItems = styled.ul<StyledSidebarNavItemsProps>(
-  ({ expanded, theme }) => ({
-    listStyle: "none",
-    maxHeight: expanded ? "75vh" : "25vh",
-    overflow: "auto",
-    // TODO(vdonato): styling
-    // * Fade in/out at the top/bottom if there is scrollable content in that
-    //   direction
-  })
+  ({ expanded, hasSidebarElements, theme }) => {
+    const expandedMaxHeight = expanded ? "75vh" : "33vh"
+    const maxHeight = hasSidebarElements ? expandedMaxHeight : null
+
+    return {
+      maxHeight,
+      listStyle: "none",
+      overflow: ["auto", "overlay"],
+      margin: 0,
+      paddingTop: theme.sizes.headerSpace,
+      paddingBottom: theme.spacing.lg,
+      // TODO(vdonato): styling
+      // * Fade in/out at the top/bottom if there is scrollable content in that
+      //   direction
```

Comment:
Can remove this comment.

Reply:
Although I guess the gradients below don't check if scrollable. Don't know if needed, though...

Reply:
Doesn't seem like it's necessary as the fade isn't visible when the container isn't overflowing/scrollable as far as I can tell.